kornia/io/io.py

```diff
@@ -54,8 +55,21 @@ def _load_image_to_tensor(path_file: Path, device: Device) -> Tensor:
 
     """
     # read image and return as `np.ndarray` with shape HxWxC
-    if path_file.suffix.lower() in [".jpg", ".jpeg"]:
-        img = kornia_rs.read_image_jpeg(str(path_file))
+    if path_file.suffix.lower() in [".jpg", ".jpeg", ".png", ".tiff"]:
+        if path_file.suffix.lower() in [".jpg", ".jpeg"]:
+            img = kornia_rs.read_image_jpegturbo(str(path_file))
+        elif path_file.suffix.lower() == ".png":
+            # TODO: implement in kornia_rs to read the headers
+            try:
+                img = kornia_rs.read_image_png_u8(str(path_file), mode="rgb")
```

Comment:
@AS1100K probably we might have to improve this interface in rust to somehow make easier to resolve the color space as some images by exposing the reading header function